tests/e2e/note-images.test.ts

```diff
@@ -0,0 +1,154 @@
+import fs from 'fs'
+import { faker } from '@faker-js/faker'
+import { expect, test } from '@playwright/test'
+import { type NoteImage, type Note } from '@prisma/client'
+import { prisma } from '#app/utils/db.server.ts'
+import { insertedUsers, loginPage } from '#tests/playwright-utils.ts'
+
+test('Users can create note with an image', async ({ page }) => {
+	const user = await loginPage({ page })
+	await page.goto(`/users/${user.username}/notes`)
+
+	const newNote = createNote()
+	const altText = 'cute koala'
+	await page.getByRole('link', { name: 'new note' }).click()
+
+	// fill in form and submit
+	await page.getByRole('textbox', { name: 'title' }).fill(newNote.title)
+	await page.getByRole('textbox', { name: 'content' }).fill(newNote.content)
+	await page.setInputFiles(
+		"input[name='images[0].file']",
+		'tests/fixtures/images/kody-notes/cute-koala.png',
+	)
+	await page.getByRole('textbox', { name: 'alt text' }).fill(altText)
+
+	await page.getByRole('button', { name: 'submit' }).click()
+	await expect(page).toHaveURL(new RegExp(`/users/${user.username}/notes/.*`))
+	await expect(page.getByRole('heading', { name: newNote.title })).toBeVisible()
+	await expect(page.getByAltText(altText)).toBeVisible()
+})
+
+test('Users can create note with multiple images', async ({ page }) => {
+	const user = await loginPage({ page })
+	await page.goto(`/users/${user.username}/notes`)
+
+	const newNote = createNote()
+	const altText1 = 'cute koala'
+	const altText2 = 'koala coder'
+	await page.getByRole('link', { name: 'new note' }).click()
+
+	// fill in form and submit
+	await page.getByRole('textbox', { name: 'title' }).fill(newNote.title)
+	await page.getByRole('textbox', { name: 'content' }).fill(newNote.content)
+	await page.setInputFiles(
+		"input[name='images[0].file']",
+		'tests/fixtures/images/kody-notes/cute-koala.png',
+	)
+	await page
+		.locator('[id="note-editor-images\\[0\\]\\.altText"]')
```

Comment:
Can we improve this locator?

Reply:
`nth()` method can be used on a `getByLabel` method to select nth textarea, like so:
```js
await page
  .getByLabel(/Alt text/i)
  .nth(0) // selects 1st textarea with label "Alt text"
  .fill(altText1)
await page
  .getByLabel(/Alt text/i)
  .nth(1) // selects 2nd textarea with label "Alt text"
  .fill(altText2)
```
Other than this, we can change the label to be different for each textarea but not sure what it should be or if it will be helpful for other than testing 🤷‍♂️

Reply:
Yes, I like that better.

Reply:
fixed in https://github.com/epicweb-dev/epic-stack/pull/420/commits/569c7a53fbc18b5fa8e15dfbb9d9ca89f8503d70